src/commands/dev.ts

```diff
@@ -31,9 +31,9 @@ import merge from 'deepmerge';
 import {EventEmitter} from 'events';
 import execa from 'execa';
 import {existsSync, promises as fs, readFileSync} from 'fs';
-import http from 'http';
+import type http from 'http';
```

Comment:
Yea, I would keep loading them all at the top. I appreciate the async perf, but since these are both Node builtins I doubt they have much impact on load performance, or at least enough impact to justify the extra complexity.